.drone.yml

```diff
@@ -0,0 +1,77 @@
+kind: pipeline
+name: test
+
+steps:
+- name: flake8
+  image: alpine/flake8:3.7.7
+  commands:
+  - flake8
+
+---
+
+kind: pipeline
+name: publish to docker hub
+
+trigger:
+  status:
+  - success
+  event:
+  - push
+  - tag
+
+depends_on:
+  - test
+
+steps:
+- name: docker push branch ${DRONE_COMMIT_BRANCH}
+  image: plugins/docker
+  settings:
+    repo: liquidinvestigations/crji-avere-europarl
+    tags: ${DRONE_COMMIT_BRANCH}
+    username:
+      from_secret: docker_username
+    password:
+      from_secret: docker_password
+  when:
+    event:
+    - push
+
+- name: docker push autotag + latest
+  image: plugins/docker
+  settings:
+    repo: liquidinvestigations/crji-avere-europarl
+    auto_tag: true
+    username:
+      from_secret: docker_username
+    password:
+      from_secret: docker_password
+
+---
+
+kind: pipeline
+name: test app
+
+depends_on:
+  - publish to docker hub
+
+steps:
+- name: test
+  image: liquidinvestigations/crji-avere-europarl:scraper
```

Comment:
please use `${DRONE_COMMIT_BRANCH}` like above, not this hard-coded tag name, so future pull requests with other tag/branch names will also work.